olm/operator-resources/operator-artifacts.jsonnet

```diff
@@ -10,9 +10,15 @@ local operatorResourcesWithVersion = std.map(addVersionToMetadataLabels, operato
 
 local addVersionToDeploymentSpec(deployment) = deployment + {
   spec+: {
-    template+: { metadata+: { labels+:
-      super.labels + { "app.kubernetes.io/version": version }
-    }}
+    template+: {
+      metadata+: {
+        labels+: super.labels + { "app.kubernetes.io/version": version }
+      },
+      spec+:
+        super.spec + {
+          containers: std.mapWithIndex(function(i, c) if i == 0 then c + {image: c.image + ":" + version} else c, super.containers)
```

Comment:
Is this the full version `0.3.9`? This is good for now - but ideally in the future we should probably create major version tags, (so, in this case `0`) and then use the major version tag here as opposed to the full version. But honestly we probably need a proper update strategy other than just "wait for the operator to crash and re-pull"... so, this is good.

Reply:
Yeah this is the full version: `image: instana/instana-agent-operator:0.3.9`